src/khoj/routers/api_chat.py

```diff
@@ -150,6 +209,66 @@ async def clear_chat_history(
     return {"status": "ok", "message": "Conversation history cleared"}
 
 
+@api_chat.post("/public/fork")
+@requires(["authenticated"])
+def fork_public_conversation(
+    request: Request,
+    common: CommonQueryParams,
+    public_conversation_slug: str,
+):
+    user = request.user.object
+
+    # Load Conversation History
+    public_conversation = PublicConversationAdapters.get_public_conversation_by_slug(public_conversation_slug)
+
+    # Duplicate Conversation History to Public Conversation
```

Comment:
nit: This comment seems incorrect. It's doing the reverse here?
```suggestion
    # Duplicate Public Conversation to Conversation History
```